### PR TITLE
REL-3256: Updated OT icons, splash screen image, and splash screen text.

### PR DESCRIPTION
**Pull request by sraaphorst**

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/resources/resources/conf/welcome.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 <p>The Observing Tool is the work of the Gemini Observatory staff including Bryan Miller, Andrew Stephens, Mischa Schirmer, and the software engineers in the Gemini Engineering Operations Department and former employees Florian Nussberger, Devin Dawson, Larry O'Brien, Nicolas Barriga, Alan Brighton, Vasudeva Upadhya, Lorraine Callahan, Darrell Denlinger, Kim Gillies, and Phil Puxley.</p>
 
-<p>The image at left is a long exposure of Gemini North during Altair laser guide star operations. Credit: Gemini Observatory.</p>
+<p>The image at left shows the interacting galaxy pair NGC 1351 and NGC 1352 imaged with the Gemini Multi-Object Spectrograph (GMOS) on the Gemini South telescope. The larger NGC 1352 is slightly more than the Milky Way while the smaller NGC 1351 is comparable to the Large Magellanic Cloud. Credit: Gemini Observatory.</p>
```
Comment (swalker2m) [inline bundle/jsky.app.ot/src/main/resources/resources/conf/welcome.html]: "is slightly more"?

Comment (sraaphorst) [reply bundle/jsky.app.ot/src/main/resources/resources/conf/welcome.html]: I thought that sounded awkward and incorrect, too, so I made a comment on the JIRA task.
I'll wait to merge this until I hear back.

Comment (arturog8m) [reply bundle/jsky.app.ot/src/main/resources/resources/conf/welcome.html]: Right - @sraaphorst asked for clarification in the REL task, so hopefully @andrewwstephens or Bryan can provide the right wording for that.

**Comment by sraaphorst** [comment]
Ah... ooops, and good call on the icon.
Fixing this now. Derp.

**Comment by sraaphorst** [comment]
Alright: text has been fixed as per Bryan's comment on the JIRA task discussion, and new commit pushed. Built, tested, and everything looks as it should.

**Comment by cquiroz** [comment]
The new icon looks vaguely familiar, are we reusing old icons?

**Comment by sraaphorst** [comment]
@cquiroz As a random curiosity, while I was waiting for a compile, I went back and checked all the old icons down to 2012A and while this one is a bit similar to two others (so I can see why you get the sense of déjà vu), they are all unique. 😸

**Pull request merged**